packages/fluentui/docs-components/.gulp.js

```diff
@@ -1 +1 @@
-module.exports = require('@fluentui/scripts/gulp/.gulp');
+module.exports = require('../../../.gulp');
```

Comment:
Can you please explain why we don't use `module.exports = require('@fluentui/scripts/gulp/.gulp');`? To avoid `@fluentui/scripts/gulp/.gulp` import or is there another reason?

Can we do `module.exports = require('@fluentui/scripts/gulp').gulpConfig` instead?

Reply:
the pattern used is based on -> global monorepo particular tool configs should live in monorepo root. 

From package perspective you would probably not export global tool configuration as part of package public api. 
Last but not least it nicely aligns with per package `gulpfile.ts` which also follows pattern mentioned in 1st sentence and adds uniformity.